src/audit.single.md

```diff
@@ -17,16 +17,20 @@ Fetch audit events for a role/resource the calling identity has `read` privilege
 |Code|Description|
 |----|-----------|
 |200|JSON list of audit events is returned|
+|401|Unauthorized|
 |403|Permission denied|
-|404|Role/resource not found|
+|404|Role/resource not found / permission denied|
 
 + Parameters
-    + kind: roles (string) - Type of object, 'roles' or 'resources'
-    + id: cucumber:host:redis001 (string) - Fully qualified ID of a Conjur role/resource, query-escaped
+    + kind: `roles` (string) - Type of object, 'roles' or 'resources'
+    + id: `cucumber:host:redis001` (string) - Fully qualified ID of a Conjur role/resource, query-escaped
```

Comment:
We say it must be query escaped, but in the example it isn't. @dustinmm80 and I have looked at this problem and unfortunately we can't find a way to fix it.